.travis.yml

```diff
@@ -1,16 +1,16 @@
+dist: latest
 language: python
 python:
-- 2.7
-- 3.5
+- 3.6
+- 3.7
 env:
-- TOXENV=django18
-- TOXENV=django19
-- TOXENV=django110
+- TOXENV=django20
```

Comment:
Wow, we actually weren't testing with Django 1.11?  Could you please add that since it's what we currently use in production? 😅

Reply:
Ouch :-) https://openedx.atlassian.net/browse/INCR-73 explicitly instructed to remove 1.10 & 1.8 support and since there was no 1.X left I trimmed python 2.7 support altogether. Which was somewhat short sighted on my part, I have to say. Working on 1.11 support now.